exercises/practice/saddle-points/.meta/config.json

```diff
@@ -0,0 +1,17 @@
+{
+  "authors": ["ryanplusplus"],
```

Comment:
`bin/configlet fmt -u` would split this over three lines.

This also applies to existing file
`exercises/practice/two-bucket/.meta/config.json`

Reply:
Great catch, I'll update all of these.